azure-pipelines.yml

```diff
@@ -491,7 +497,7 @@ jobs:
           PR_BRANCH: $(pr.branch)
 
   - job: notify_user
-    condition: always()
+    condition: eq(variables['Build.Reason'], 'PullRequest')
```

Comment:
This job doesn't really make sense on master commits as there is no followup action required — and build failures are already reported.